apps/perf-test/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "perf-test",
+  "description": "Sample todo app using office-ui-fabric-react.",
```

Comment:
```suggestion
  "description": "Sandbox app for analyzing component performance.",
```

Or similar.. tho it doesn't really matter

Reply:
I'm also in favor of accurate comments, even in unpublished projects :)